### DISCUSSION TITLE
HAProxy and multiple configuration files

### DISCUSSION THREAD

**Discussion by netbja** [roxy-wi/roxy-wi#349] [Q&A] [answered]
Dear all,

We have a lot of HAProxy and we use multiple config files, global.cfg, tcp.cfg, http.cfg, users.cfg, it's easier to maintain.
But did you think it's possible to add this split config with roxy-wi ?

Thanks !

**Comment by Aidaho12** [comment, marked as answer]
Yes, it possible

**Reply by netbja** [reply]
Great !
